virttest/utils_config.py

```diff
@@ -463,3 +463,19 @@ class LibvirtGuestsConfig(LibvirtConfigCommon):
         'BYPASS_CACHE': 'boolean',
         'SYNC_TIME': 'boolean',
     }
+
+
+class LibvirtSanLockConfig(LibvirtConfigCommon):
+
+    """
+    Class for libvirt san lock config file.
+    """
+    conf_path = '/etc/libvirt/qemu-sanlock.conf'
+    __option_types__ = {
+        'auto_disk_leases': 'boolean',
+        'disk_lease_dir': 'string',
+        'host_id': 'int',
+        'require_lease_for_disks': 'boolean',
```

Comment:
`auto_disk_leases` and `require_lease_for_disks` looks to be boolean from libvirt source, but as part of configuration, it asks to set as 1 to enable, should be consider it as int for handling from configuration file.

Reply:
Yes, somehow you are right. But since config file doesn't support True or False value, so it need set to "1" or "0". But from user point of view(libvirt), boolean is more meaningful ,and actually it will transfer to 1 if True was set . I think on other config such as qemu.conf ,libvirtd are also the same.

Reply:
cool then 👍